views/lang/latin/verb/latin-verb-irregular.js

```diff
@@ -0,0 +1,72 @@
+import { Constants, LanguageModelFactory, Feature } from 'alpheios-data-models'
+import LatinView from '../latin-view.js'
+
+import GroupFeatureType from '../../../lib/group-feature-type'
+import Form from '../../../../lib/form.js'
+import Table from '../../../lib/table'
+
+export default class LatinVerbIrregularView extends LatinView {
+  constructor (inflectionData, locale) {
+    super(inflectionData, locale)
+
+    this.id = 'verbConjugationIrregular'
+    this.name = 'verb-irregular'
+    this.title = 'Verb Conjugation (Irregular)'
+
+    const lemmaValues = this.dataset.getVerbsIrregularLemma(inflectionData.targetWord)
+    let lemmasType = new Feature(Feature.types.hdwd, lemmaValues, LatinVerbIrregularView.languageID)
+
+    this.features = {
+      lemmas: new GroupFeatureType(lemmasType, 'Lemma'),
+      tenses: new GroupFeatureType(this.language_features[Feature.types.tense], 'Tenses'),
+      numbers: new GroupFeatureType(this.language_features[Feature.types.number], 'Number'),
+      persons: new GroupFeatureType(this.language_features[Feature.types.person], 'Person'),
+      moods: new GroupFeatureType(this.language_features[Feature.types.mood], 'Mood')
+    }
+    this.createTable()
+  }
+
+  createTable () {
+    this.table = new Table([this.features.lemmas, this.features.moods, this.features.tenses, this.features.numbers, this.features.persons])
+    let features = this.table.features
+    features.columns = [ this.features.lemmas, this.features.moods ]
```

Comment:
We need to add a column grouping on voice. Not all of the forms have it, but it if it's present we need to group on it.  E.g. this is why you are getting more values in the cells in the table for "fero" than are there in the V1 version. There are different values for active and passive voice.